pkg/cmd/create/create_cluster_gke.go

```diff
@@ -162,6 +176,18 @@ func (o *CreateClusterGKEOptions) Run() error {
 }
 
 func (o *CreateClusterGKEOptions) createClusterGKE() error {
+	configFile := o.InstallOptions.Flags.ConfigFile
```

Comment:
Love this PR!  I wonder if we should move the config file flag and this section of code down into install options or something so it's reusable for all create cluster and install commands?

Reply:
Probably all the way down to common options and the root command, so that any command can make use of this. I am also in the process of using Viper as part of binding environment variables to command options for `step create ` commands. So far I am using Viper just in the step commands directly similar to this code, but I was already wondering how this could be made more generic and usable for other commands.